operate/services/manage.py

```diff
@@ -1902,9 +1902,10 @@ async def funding_job(
     def deploy_service_locally(
         self,
         service_config_id: str,
-        force: bool = True,
```

Comment:
let's keep it for now, because we don't know if we'll need it in the future. Also, not needed atm but please update the docstring also when changing the parameters.

Reply:
avoiding for now, otherwise too many variables